action.yml

```
@@ -7,14 +7,16 @@ branding:
   color: 'gray-dark'
 
 inputs:
-  inline:
-    description: 'Value passed to the --inline flag. Options: yes | no. Default value: yes.'
-    default: 'yes'
   coverage:
     description: 'Value determining whether to test with coverage or not. Options: true | false. Default value: true.'
     default: 'true'
   depwarn:
     description: 'Value passed to the --depwarn flag. Options: yes | no | error. Default value: yes.'
+  force_latest_compat:
+    description: 'If true, then, for each `[compat]` entry in the active project, only the latest compatible version will be allowed. Options: true | false | autodetect. Default value: auto. If the value is auto, then force_latest_compat will be set to true if this is a Dependabot/CompatHelper pull request, and false otherwise.'
```

```suggestion
    description: 'If true, then, for each `[compat]` entry in the active project, only the latest compatible version will be allowed. Options: true | false | auto. Default value: auto. If the value is auto, then force_latest_compat will be set to true if this is a Dependabot/CompatHelper pull request, and false otherwise.'
```

Sorry, I missed one.